bedrock/base/templates/includes/protocol/footer/footer.html

```diff
@@ -6,6 +6,130 @@
 
 {% set utm_params = 'utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=footer' %}
 
+{% if switch('brand-refresh') %}
+  {% block page_css %}
+    {{ css_bundle('footer-refresh') }}
+  {% endblock %}
```

Comment:
**Nit:** we generally try to keep CSS files in the `<head>` of the page to avoid flashes of un-styled content. Global CSS like this should probably go in the `site_css` block in our base templates:

- https://github.com/mozilla/bedrock/blob/main/bedrock/base/templates/base-protocol.html#L74-L76
- https://github.com/mozilla/bedrock/blob/main/bedrock/base/templates/base-protocol-mozilla.html#L11-L13

Also, rather than create a separate CSS file for each page element, it might be slightly more performant if we created a new [base CSS bundle](https://github.com/mozilla/bedrock/blob/main/media/css/protocol/protocol-mozilla.scss) that includes the new global elements, such as the footer and navigation, and excludes the CSS for the old elements (to save redundant styles being loaded)?